final_assessment/task_2.py

```diff
@@ -0,0 +1,37 @@
+# TASK 2
+#
+# Given an index limit, find all corresponding Fibonacci values up to that limit in a sequence
+# and return the sum of all even Fibonacci numbers. See more details in the task description in
+# your assessment paper.
+
+fn = (0,1,1,2,3,5,8,13,21,34,55,89,144,233,377,610,987,1597,2584,4181,6765)
```

Comment:
5/25 - Good attempt here, however logic errors (mainly in line 13) meant that no test cases pass and that the function does not work as expected.